source/index.js

```diff
@@ -21,5 +22,13 @@ var loop = new Afloop(function(delta) {
     game.children.forEach((child) => {
         child.update(delta)
     })
+    if (game.playerCount === 0) {
```

Comment:
Ha! Way to use triple equality. If you hadn't, javascript would parse anything "falsy" to be equal to zero, because javascript is weeeeird.